Bindings/Java/Matlab/Hopper_Device/OptimizeHopper.m

```diff
@@ -55,35 +55,15 @@
         lowerBound = 0.25*ones(numVars, 1); % seconds
         upperBound = 0.75*ones(numVars, 1); % seconds
         options = optimoptions('ga','FunctionTolerance', 1e-2, ...
-                                    'MaxGenerations', 10, ...
+                                    'MaxGenerations', 1, ...
```

Comment:
You can still get 3 nice hops with 1 generation?

Reply:
I ran it a few times with one generation, and I'm still getting solid triple jumps (close to 1.75 if not above). So we could probably get away with one generation.